MAP/src/components/TimeSlider/TimeSlider.js

```diff
@@ -626,7 +626,7 @@ const TimeSlider = (props) => {
           value={currentDateValue}
         />
         <span title="Select End Date" className={`last ${tooltipCss}`}>
-          {toSliderString(new Date(lastDay), 'en')}
+          {toSliderString(new Date(lastDay? lastDay : null), 'en')}
```

Comment:
@YehualashetGit I had to make these changes to get storybook to work without error - can you check that there isn't a better way for dealing with this

Reply:
Ohh okay thanks for the fix, I was planning to fix it, yeah this fix is good, even though it only appears in the storybook .yea we are good to go